job_templates/ci_job.xml.em

```diff
@@ -229,70 +241,88 @@ setlocal enableDelayedExpansion
 rmdir /S /Q ws workspace "work space"
 
 echo "# BEGIN SECTION: Determine arguments"
-set "PATH=%PATH:"=%"
-set "CI_ARGS=--force-ansi-color --workspace-path %WORKSPACE%"
-if "%CI_BRANCH_TO_TEST%" NEQ "" (
-  set "CI_ARGS=%CI_ARGS% --test-branch %CI_BRANCH_TO_TEST%"
+set "PATH=!PATH:"=!"
+set "CI_ARGS=--force-ansi-color --workspace-path !WORKSPACE!"
+if "!CI_BRANCH_TO_TEST!" NEQ "" (
+  set "CI_ARGS=!CI_ARGS! --test-branch !CI_BRANCH_TO_TEST!"
```

Comment:
As the code stands now, you are right in that it is not strictly necessary for the vast majority of them (CI_ARGS is the obvious one that needs to be !...!).  My motivation for converting them all is that this whole %...% vs. !...! business isn't really intuitive, particularly in the Jenkins environment.  If someone ever tries to make another change where they change the value of the variable, they'll have to re-discover the correct mechanism.  By switching them all, following the pattern becomes easier, and the variables will more-or-less always do what you expect.

That being said, if you still think we should leave them as %...%, I can switch it back (except for the ones that need to be !...!).  Thoughts?